src/components/TopBar/components/UserMenu/context/Modifier.tsx

```diff
@@ -22,7 +22,7 @@ class Modifier extends React.Component<ComposedProps, {}> {
     return null;
   }
 
-  state = {};
+  state: any = {};
```

Comment:
We can probably 🔥 this and change line 14 to `never`

Reply:
I've got no idea what this does. So I elected for not changing the code and just making typescript happy. It looks like this component still has state it's just defined in the passed setMobileUserMenuProps function.

Thinking a little deeper, matching this type with the second arg of `React.Component<ComposedProps, {}>` is a better plan. I've changed it to `state: {}`.